tests/lib/constants/constants.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, code, status, title, detail, links={}):
         self.title = title
         self.detail = detail
         self.links = {
-            **{"about": "https://digital.nhs.uk/developer/api-catalogue/communications-manager"},
+            **{"about": "https://digital.nhs.uk/developer/api-catalogue/nhs-notify"},
```

Comment:
This is out of scope for documentation and took a bit of a sledgehammer to the tests https://dev.azure.com/NHSD-APIM/API%20Platform/_build/results?buildId=238606&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=5ddaae21-daff-5db5-bedd-6664f551df01

Reply:
Have reverted this change

Reply:
For reasons I don't understand I had to unrevert this change to fix the tests?!